examples/graphics/mesh-morph.html

```diff
@@ -132,15 +132,15 @@
             model.morphInstances.push(morphInstance);
 
             // Create Entity and add it to the scene
-            this.entity = new pc.Entity();
-                entity.setLocalPosition(x, y, z);
-            app.root.addChild(this.entity);
+            var entity = new pc.Entity();
+            entity.setLocalPosition(x, y, z);
+            app.root.addChild(entity);
 
             // Add a model compoonent
-            app.systems.model.addComponent(this.entity, {
+            app.systems.model.addComponent(entity, {
```

Comment:
Whoops - this is the old style. Can you change to:
```
entity.addComponent('model', {
    type: 'asset'
});
```

Reply:
Also, does this work:
```
entity.addComponent('model', {
    model: model,
    type: 'asset'
});
```

I have a feeling there's a bug that makes that not work, but it would be cleaner code-wise.

Reply:
changed to the first one, the second does not work.